### ISSUE TITLE
move support requests from npm/npm to this repo

### ISSUE THREAD

**Issue by linclark** [npm/support-cli#4]
One of the Google folks recently posted a [tool to move issues](https://github-issue-mover.appspot.com/) from one repo to another. It preserves the comments, and posts a comment about how the issue has been moved.

It might not be the best solution, since you have to move issues one at a time, but might be adequate since this is a one time job.

**Comment by faiq** [comment]
I think its a good suggestion. The only opposition to that would be it would cause some confusion.
Would it be so bad to keep them on npm/npm until this and `npm report` are ready, and once they are ready tell people to come here and comment.

**Comment by othiym23** [comment]
> Would it be so bad to keep them on npm/npm until this and npm report are ready, and once they are ready tell people to come here and comment?

No, no it would not. Another thing to consider is that this migration will cause a _lot_ of notifications churn for people watching one or both of the repos, so we probably should do this in such a way that people can do a one-time reset of their notification support to ditch the huge volume of changes.
